app/view/twig/files/_files.twig

```diff
@@ -12,7 +12,7 @@
 
     <tbody>
         {% for file in context.files %}
-            <tr>
+            <tr class="entry" data-bolt-browse-name="{{ file.filename|excerpt(80)|shy | trim }}">
```

Comment:
I don't think any of these filters are needed.
Why limit the filename?
The `|shy` filter isn't needed as this is data not display text.
The `|trim` filter isn't needed as filenames cannot contain spaces.